### PR TITLE
Update documentation on known issues

### PR DESCRIPTION
**Pull request by rouille**
### Purpose
Update documentation on known issues. Numbers quoted and code snippets have been updated with latest 2022 pipeline outputs. A section has been added for global extreme screening.

### What the code is doing
N/A

### Testing
N/A

### Where to look
The global extreme is new and a review by a native english speaker would be apprecited

### Usage Example/Visuals
N/A

### Review estimate
5min

### Future work
N/A

### Checklist
- [x] Update the documentation to reflect changes made in this PR
- [x] Format all updated python files using `black`
- [x] Clear outputs from all notebooks modified
- [x] Add docstrings and type hints to any new functions created

### CODE REVIEW

File: docs/docs/Data Validation/Known Issues.md [hunk 1]
```diff
@@ -9,154 +9,187 @@ We maintain a [list of these known issues on GitHub](https://github.com/singular
...
-54724                       0.00                  0.0                                  0.0
-54809                    9033.11            1040724.0                             114648.0
-58256                     666.00               2274.0                               2274.0
-59817                    1700.00               5802.0                               5802.0
-59825                       0.00                  0.0                                  0.0
-65498                       0.00                  0.0                                  0.0
+For 7 plants, the total generation or fuel allocated to individual generators does not match the total generation or fuel associated with the plant in the raw input data. Possible causes include inconsistent reporting of prime movers or fuel types between EIA-923 tables, mid-year plant retirements, anomalous fuel totals being reported, or other unknown issues. 
+
+```log
+2024-04-03 12:47:46,436 [WARNING] oge.oge.validation:128 There are 7 plant/fuel combinations with non-zero (missing) net generation or fuel consumed that are different in aloocated EIA-923 and EIA-923 Input
```
Comment (grgmiller) [inline docs/docs/Data Validation/Known Issues.md:38]: "aloocated" is mis-spelled here. We should update this in the logger text.

File: docs/docs/Data Validation/Known Issues.md [hunk 2]
```diff
@@ -9,154 +9,187 @@ We maintain a [list of these known issues on GitHub](https://github.com/singular
...
+9             971                                 DFO                 NG                                    petroleum                 natural_gas    MISO
 [...]
 ```
 
 ### Missing NOx and SO2 emission factors for Fuel Cells
 Our validation check identifies that there are missing NOx and SO2 emission factors for plants with fuel cell prime movers. This is a known and expected issue - to our knowledge emission factors for fuel cells have not yet been published.
 
+
+### Global extreme are detected in CEMS timeseries
+The CEMS gross generation, fuel consumed and CO2 emission timeseries are screened for global extremes in a validation check. The check reports the number of hours where a value exceeds the median value by at least 10 times together with the mean deviation from the median of all the global extremes.
```
Comment (grgmiller) [inline docs/docs/Data Validation/Known Issues.md:99]: Suggested edit:
We identify where gross generation, fuel consumption, and CO2 emission timeseries in the reported CEMS data may be anomalous. The check reports the number of observations where the hourly value exceeds the median value for the unit by a factor of 10 or more, as well as the mean deviation of these extreme values from the median.

**Review by grgmiller — APPROVED** [review]
Looks good, a few small things

File: docs/docs/Data Validation/Known Issues.md (hunk 1 above)
Comment (rouille) [reply docs/docs/Data Validation/Known Issues.md]: Done

File: docs/docs/Data Validation/Known Issues.md (hunk 2 above)
Comment (rouille) [reply docs/docs/Data Validation/Known Issues.md]: Done

**Pull request merged**